### ISSUE TITLE
Updated version of Django Q: Django Q2

### ISSUE THREAD

**Issue by GDay** [Koed00/django-q#701]
I have been using library a lot and was a bit concerned that it hasn't been updated in over a year. Therefore, I have created a fork in which I have merged a few of the current open PRs. I have also pushed a few patches myself.

The new repo/package: https://github.com/GDay/django-q2. 
It's compatible with `django-q`, you don't need to change any code to make this work. Just swap the libraries.

What I have done so far:
- Updated all dependencies. Dropped support for Django 2.2 (it should still work though) and added support for the newer Python versions and Django 4.1.x. 
- Removed Disque as it's outdated and now part of Redis as a submodule instead.
- I have removed the `Arrow`, `Redis` and `Blessed` dependency. It currently only needs `django` and `django-picklefield` to work. 
   - If you need the monitor commands, then you will need to install `Blessed` as well.
- Merged https://github.com/Koed00/django-q/pull/677 to show function name in the logs as it runs (next to the unique id/human hash)
- Moved from Codecov to coveralls. Codecov tends to fail quite often at random, while coveralls has been very stable.
- Moved the docs to https://django-q2.readthedocs.io (and updated them)
- Merged https://github.com/Koed00/django-q/pull/674 which adds Turkish translations
- Automated publishing to pypi.
- Merged https://github.com/Koed00/django-q/pull/604 to fix a connection issue
- Merged https://github.com/Koed00/django-q/pull/603 to fix a potential security issue
- Merged https://github.com/Koed00/django-q/pull/605 to improve the admin area
- Merged https://github.com/Koed00/django-q/pull/687 to improve saving tasks
- Made testing a lot easier with docker-compose

And perhaps a few more things that I forgot.

There are a few more bugs that I have encountered with `Django Q` that I am going to solve over the next few days and I plan to keep the Q2 library up to date for the foreseeable future. I am also open to adding a few collaborators to make sure the project stays alive.

Also, it looks like Ilan (the author of this project) has replied a few times in this thread: https://github.com/Koed00/django-q/issues/664. If this project does indeed get brought back to life in the near future (with more maintainers), then I am happy to archive my repo and create PRs for the changes that I have made myself back to this project.

Let me know if there are any questions.

**Comment by PsychicSmurf** [comment]
This is awesome. Thank you so much for picking this up.

**Comment by jimjag** [comment]
Agreed. If you call out for additional developers/contributors/collaborators/maintainers, I volunteer.

**Comment by GDay** [comment]
@jimjag thanks! I have just added you as a collaborator. It might be better to move my repo to an organization to get some more fine-grained permission levels.

**Comment by jimjag** [comment]
Not a bad idea (about the org)...
